##  16.5 Eukaryotic Post-transcriptional Gene Regulation 

### Summary

By the end of this section, you will be able to: 

  - Understand RNA splicing and explain its role in regulating gene expression
  - Describe the importance of RNA stability in gene regulation

RNA is transcribed, but must be processed into a mature form before translation can begin. This processing after an RNA molecule has been transcribed, but before it is translated into a protein, is called post-transcriptional modification. As with the epigenetic and transcriptional stages of processing, this post-transcriptional step can also be regulated to control gene expression in the cell. If the RNA is not processed, shuttled, or translated, then no protein will be synthesized.

### RNA splicing, the first stage of post-transcriptional control

In eukaryotic cells, the RNA transcript often contains regions, called introns, that are removed prior to translation. The regions of RNA that code for protein are called exons (Figure). After an RNA molecule has been transcribed, but prior to its departure from the nucleus to be translated, the RNA is processed and the introns are removed by splicing.

![A pre-mRNA has four exons separated by three introns. The pre-mRNA can be alternatively spliced to create two different proteins, each with three exons. One protein contains exons one, two, and three. The other protein contains exons one, three and four.][1] _Pre-mRNA can be alternatively spliced to create different proteins._Evolution Connection

Alternative RNA SplicingIn the 1970s, genes were first observed that exhibited alternative RNA splicing. Alternative RNA splicing is a mechanism that allows different protein products to be produced from one gene when different combinations of introns, and sometimes exons, are removed from the transcript (Figure). This alternative splicing can be haphazard, but more often it is controlled and acts as a mechanism of gene regulation, with the frequency of different splicing alternatives controlled by the cell as a way to control the production of different protein products in different cells or at different stages of development. Alternative splicing is now understood to be a common mechanism of gene regulation in eukaryotes; according to one estimate, 70 percent of genes in humans are expressed as multiple proteins through alternative splicing.

![Diagram shows five methods of alternative splicing of pre-mRNA. When exon skipping occurs, an exon is spliced out in one mature mRNA product and retained in another. When mutually exclusive exons are present in the pre-mRNA, only one is retained in the mature mRNA. When an alternative 5’ donor site is present, the location of the 5’ splice site is variable. When an alternative 3’ acceptor site is present, the location of the 3’ splice site is variable. Intron retention results in an intron being retained in one mature mRNA and spliced out in another.][2] _There are five basic modes of alternative splicing._

How could alternative splicing evolve? Introns have a beginning and ending recognition sequence; it is easy to imagine the failure of the splicing mechanism to identify the end of an intron and instead find the end of the next intron, thus removing two introns and the intervening exon. In fact, there are mechanisms in place to prevent such intron skipping, but mutations are likely to lead to their failure. Such “mistakes” would more than likely produce a nonfunctional protein. Indeed, the cause of many genetic diseases is alternative splicing rather than mutations in a sequence. However, alternative splicing would create a protein variant without the loss of the original protein, opening up possibilities for adaptation of the new variant to new functions. Gene duplication has played an important role in the evolution of new functions in a similar way by providing genes that may evolve without eliminating the original, functional protein.

### Control of RNA Stability

Before the mRNA leaves the nucleus, it is given two protective "caps" that prevent the end of the strand from degrading during its journey. The **5' cap**, which is placed on the 5' end of the mRNA, is usually composed of a methylated guanosine triphosphate molecule (GTP). The **poly-A tail**, which is attached to the 3' end, is usually composed of a series of adenine nucleotides. Once the RNA is transported to the cytoplasm, the length of time that the RNA resides there can be controlled. Each RNA molecule has a defined lifespan and decays at a specific rate. This rate of decay can influence how much protein is in the cell. If the decay rate is increased, the RNA will not exist in the cytoplasm as long, shortening the time for translation to occur. Conversely, if the rate of decay is decreased, the RNA molecule will reside in the cytoplasm longer and more protein can be translated. This rate of decay is referred to as the RNA stability. If the RNA is stable, it will be detected for longer periods of time in the cytoplasm.

Binding of proteins to the RNA can influence its stability. Proteins, called **RNA-binding proteins**, or RBPs, can bind to the regions of the RNA just upstream or downstream of the protein-coding region. These regions in the RNA that are not translated into protein are called the **untranslated regions**, or UTRs. They are not introns (those have been removed in the nucleus). Rather, these are regions that regulate mRNA localization, stability, and protein translation. The region just before the protein-coding region is called the **5' UTR**, whereas the region after the coding region is called the **3' UTR** (Figure). The binding of RBPs to these regions can increase or decrease the stability of an RNA molecule, depending on the specific RBP that binds.

![In the mature RNA molecule, exons are spliced together between the 5' and 3' untranslated regions. A 5' cap is attached to the 5' untranslated region, and a poly-A tail is attached to the 3' untranslated region. RNA-binding proteins associate with the 5' and 3' untranslated regions.][3] _The protein-coding region of mRNA is flanked by 5' and 3' untranslated regions (UTRs). The presence of RNA-binding proteins at the 5' or 3' UTR influences the stability of the RNA molecule._

#### RNA Stability and microRNAs

In addition to RBPs that bind to and control (increase or decrease) RNA stability, other elements called microRNAs can bind to the RNA molecule. These **microRNAs**, or miRNAs, are short RNA molecules that are only 21–24 nucleotides in length. The miRNAs are made in the nucleus as longer pre-miRNAs. These pre-miRNAs are chopped into mature miRNAs by a protein called **dicer**. Like transcription factors and RBPs, mature miRNAs recognize a specific sequence and bind to the RNA; however, miRNAs also associate with a ribonucleoprotein complex called the **RNA-induced silencing complex (RISC)**. RISC binds along with the miRNA to degrade the target mRNA. Together, miRNAs and the RISC complex rapidly destroy the RNA molecule.

### Section Summary

Post-transcriptional control can occur at any stage after transcription, including RNA splicing, nuclear shuttling, and RNA stability. Once RNA is transcribed, it must be processed to create a mature RNA that is ready to be translated. This involves the removal of introns that do not code for protein. Spliceosomes bind to the signals that mark the exon/intron border to remove the introns and ligate the exons together. Once this occurs, the RNA is mature and can be translated. RNA is created and spliced in the nucleus, but needs to be transported to the cytoplasm to be translated. RNA is transported to the cytoplasm through the nuclear pore complex. Once the RNA is in the cytoplasm, the length of time it resides there before being degraded, called RNA stability, can also be altered to control the overall amount of protein that is synthesized. The RNA stability can be increased, leading to longer residency time in the cytoplasm, or decreased, leading to shortened time and less protein synthesis. RNA stability is controlled by RNA-binding proteins (RPBs) and microRNAs (miRNAs). These RPBs and miRNAs bind to the 5' UTR or the 3' UTR of the RNA to increase or decrease RNA stability. Depending on the RBP, the stability can be increased or decreased significantly; however, miRNAs always decrease stability and promote decay.

### Glossary

**3' UTR** 3' untranslated region; region just downstream of the protein-coding region in an RNA molecule that is not translated

**5' cap**a methylated guanosine triphosphate (GTP) molecule that is attached to the 5' end of a messenger RNA to protect the end from degradation

**5' UTR** 5' untranslated region; region just upstream of the protein-coding region in an RNA molecule that is not translated

**dicer** enzyme that chops the pre-miRNA into the mature form of the miRNA

**microRNA (miRNA)** small RNA molecules (approximately 21 nucleotides in length) that bind to RNA molecules to degrade them

**poly-A tail**a series of adenine nucleotides that are attached to the 3' end of an mRNA to protect the end from degradation

**RNA-binding protein (RBP)** protein that binds to the 3' or 5' UTR to increase or decrease the RNA stability

**RNA stability** how long an RNA molecule will remain intact in the cytoplasm

**untranslated region** segment of the RNA molecule that are not translated into protein. These regions lie before (upstream or 5') and after (downstream or 3') the protein-coding region

**RISC** protein complex that binds along with the miRNA to the RNA to degrade it

   [1]: https://cnx.org/resources/becb64c82af9cdb46aaf0879035b9c038425e930/Figure_16_05_03.jpg
   [2]: https://cnx.org/resources/0563d586bf7b1702af2bff4abc194a92c1eafea0/Figure_15_04_02.jpg
   [3]: https://cnx.org/resources/59f6367c9a484f9556ff2e26d14eb080e4a82594/Figure_16_05_02.jpg

