##  15.2 Prokaryotic Transcription 

### Summary

By the end of this section, you will be able to do the following: 

  - List the different steps in prokaryotic transcription
  - Discuss the role of promoters in prokaryotic transcription
  - Describe how and when transcription is terminated

The prokaryotes, which include Bacteria and Archaea, are mostly single-celled organisms that, by definition, lack membrane-bound nuclei and other organelles. A bacterial chromosome is a closed circle that, unlike eukaryotic chromosomes, is not organized around histone proteins. The central region of the cell in which prokaryotic DNA resides is called the nucleoid region. In addition, prokaryotes often have abundant **plasmids**, which are shorter, circular DNA molecules that may only contain one or a few genes. Plasmids can be transferred independently of the bacterial chromosome during cell division and often carry traits such as those involved with antibiotic resistance.

Transcription in prokaryotes (and in eukaryotes) requires the DNA double helix to partially unwind in the region of mRNA synthesis. The region of unwinding is called a **transcription bubble.** Transcription always proceeds from the same DNA strand for each gene, which is called the **template strand**. The mRNA product is complementary to the template strand and is almost identical to the other DNA strand, called the **nontemplate strand**, or the coding strand. The only nucleotide difference is that in mRNA, all of the T nucleotides are replaced with U nucleotides (Figure). In an RNA double helix, A can bind U via two hydrogen bonds, just as in A–T pairing in a DNA double helix.

![Illustration shows R N A synthesis by R N A polymerase. The R N A strand is synthesized in the 5 prime to 3 prime direction.][1] _Messenger RNA is a copy of protein-coding information in the coding strand of DNA, with the substitution of U in the RNA for T in the coding sequence. However, new RNA nucleotides base pair with the nucleotides of the template strand. RNA is synthesized in its 5'-3' direction, using the enzyme RNA polymerase. As the template is read, the DNA unwinds ahead of the polymerase and then rewinds behind it._

The nucleotide pair in the DNA double helix that corresponds to the site from which the first 5' mRNA nucleotide is transcribed is called the +1 site, or the **initiation site**. Nucleotides preceding the initiation site are denoted with a “-” and are designated _upstream nucleotides_. Conversely, nucleotides following the initiation site are denoted with “+” numbering and are called _downstream nucleotides_.

### Initiation of Transcription in Prokaryotes

Prokaryotes do not have membrane-enclosed nuclei. Therefore, the processes of transcription, translation, and mRNA degradation can all occur simultaneously. The intracellular level of a bacterial protein can quickly be amplified by multiple transcription and translation events that occur concurrently on the same DNA template. Prokaryotic genomes are very compact, and prokaryotic transcripts often cover more than one gene or cistron (a coding sequence for a single protein). Polycistronic mRNAs are then translated to produce more than one kind of protein.

Our discussion here will exemplify transcription by describing this process in _Escherichia coli_, a well-studied eubacterial species. Although some differences exist between transcription in _E. coli_ and transcription in archaea, an understanding of _E. coli _transcription can be applied to virtually all bacterial species.

#### Prokaryotic RNA Polymerase

Prokaryotes use the same RNA polymerase to transcribe all of their genes. In _E. coli_, the polymerase is composed of five polypeptide subunits, two of which are identical. Four of these subunits, denoted _α_, _α_, _β_, and _β_', comprise the polymerase **core enzyme**. These subunits assemble every time a gene is transcribed, and they disassemble once transcription is complete. Each subunit has a unique role; the two _α_-subunits are necessary to assemble the polymerase on the DNA; the _β_-subunit binds to the ribonucleoside triphosphate that will become part of the nascent mRNA molecule; and the _β_' subunit binds the DNA template strand. The fifth subunit, _σ_, is involved only in transcription initiation. It confers transcriptional specificity such that the polymerase begins to synthesize mRNA from an appropriate initiation site. Without _σ_, the core enzyme would transcribe from random sites and would produce mRNA molecules that specified protein gibberish. The polymerase comprised of all five subunits is called the **holoenzyme**.

#### Prokaryotic Promoters

A **promoter** is a DNA sequence onto which the transcription machinery, including RNA polymerase, binds and initiates transcription. In most cases, promoters exist upstream of the genes they regulate. The specific sequence of a promoter is very important because it determines whether the corresponding gene is transcribed all the time, some of the time, or infrequently. Although promoters vary among prokaryotic genomes, a few elements are evolutionarily conserved in many species. At the -10 and -35 regions upstream of the initiation site, there are two _promoter consensus sequences_, or regions that are similar across all promoters and across various bacterial species (Figure). The -10 sequence, called the -10 region, has the consensus sequence TATAAT. The -35 sequence has the consensus sequence TTGACA. These consensus sequences are recognized and bound by _σ_. Once this interaction is made, the subunits of the core enzyme bind to the site. The A–T-rich -10 region facilitates unwinding of the DNA template, and several phosphodiester bonds are made. The transcription initiation phase ends with the production of abortive transcripts, which are polymers of approximately 10 nucleotides that are made and released.

![Illustration shows the sigma subunit of R N A polymerase bound to two consensus sequences that are 10 and 35 bases upstream of the transcription start site. R N A polymerase is bound to sigma..][2] _The _σ_ subunit of prokaryotic RNA polymerase recognizes consensus sequences found in the promoter region upstream of the transcription start site. The _σ_ subunit dissociates from the polymerase after transcription has been initiated._

### Elongation and Termination in Prokaryotes

The transcription elongation phase begins with the release of the _σ_ subunit from the polymerase. The dissociation of _σ_ allows the core enzyme to proceed along the DNA template, synthesizing mRNA in the 5' to 3' direction at a rate of approximately 40 nucleotides per second. As elongation proceeds, the DNA is continuously unwound ahead of the core enzyme and rewound behind it. The base pairing between DNA and RNA is not stable enough to maintain the stability of the mRNA synthesis components. Instead, the RNA polymerase acts as a stable linker between the DNA template and the nascent RNA strands to ensure that elongation is not interrupted prematurely.

### Prokaryotic Termination Signals

Once a gene is transcribed, the prokaryotic polymerase needs to be instructed to dissociate from the DNA template and liberate the newly made mRNA. Depending on the gene being transcribed, there are two kinds of termination signals. One is protein-based and the other is RNA-based. **Rho-dependent termination** is controlled by the rho protein, which tracks along behind the polymerase on the growing mRNA chain. Near the end of the gene, the polymerase encounters a run of G nucleotides on the DNA template and it stalls. As a result, the rho protein collides with the polymerase. The interaction with rho releases the mRNA from the transcription bubble.

**Rho-independent termination** is controlled by specific sequences in the DNA template strand. As the polymerase nears the end of the gene being transcribed, it encounters a region rich in C–G nucleotides. The mRNA folds back on itself, and the complementary C–G nucleotides bind together. The result is a **stable hairpin** that causes the polymerase to stall as soon as it begins to transcribe a region rich in A–T nucleotides. The complementary U–A region of the mRNA transcript forms only a weak interaction with the template DNA. This, coupled with the stalled polymerase, induces enough instability for the core enzyme to break away and liberate the new mRNA transcript.

Upon termination, the process of transcription is complete. By the time termination occurs, the prokaryotic transcript would already have been used to begin synthesis of numerous copies of the encoded protein because these processes can occur concurrently. The unification of transcription, translation, and even mRNA degradation is possible because all of these processes occur in the same 5' to 3' direction, and because there is no membranous compartmentalization in the prokaryotic cell (Figure). In contrast, the presence of a nucleus in eukaryotic cells precludes simultaneous transcription and translation.

![Illustration shows multiple m R N As transcribed off one gene. Ribosomes attach to the m R N A before transcription is complete and begin to make protein.][3] _Multiple polymerases can transcribe a single bacterial gene while numerous ribosomes concurrently translate the mRNA transcripts into polypeptides. In this way, a specific protein can rapidly reach a high concentration in the bacterial cell._

### Section Summary

In prokaryotes, mRNA synthesis is initiated at a promoter sequence on the DNA template comprising two consensus sequences that recruit RNA polymerase. The prokaryotic polymerase consists of a core enzyme of four protein subunits and a _σ_ protein that assists only with initiation. Elongation synthesizes mRNA in the 5' to 3' direction at a rate of 40 nucleotides per second. Termination liberates the mRNA and occurs either by rho protein interaction or by the formation of an mRNA hairpin.

### Critical Thinking Questions

If mRNA is complementary to the DNA template strand and the DNA template strand is complementary to the DNA nontemplate strand, then why are base sequences of mRNA and the DNA nontemplate strand not identical? Could they ever be?

DNA is different from RNA in that T nucleotides in DNA are replaced with U nucleotides in RNA. Therefore, they could never be identical in base sequence.

In your own words, describe the difference between rho-dependent and rho-independent termination of transcription in prokaryotes.

Rho-dependent termination is controlled by the rho protein, which tracks along behind the polymerase on the growing mRNA chain. Near the end of the gene, the polymerase stalls at a run of G nucleotides on the DNA template. The rho protein collides with the polymerase and releases mRNA from the transcription bubble. Rho-independent termination is controlled by specific sequences in the DNA template strand. As the polymerase nears the end of the gene being transcribed, it encounters a region rich in C–G nucleotides. This creates an mRNA hairpin that causes the polymerase to stall right as it begins to transcribe a region rich in A–T nucleotides. Because A–U bonds are less thermostable, the core enzyme falls away.

A fragment of bacterial DNA reads: 

3’ –TACCTATAATCTCAATTGATAGAAGCACTCTAC– 5’ 

Assuming that this fragment is the template strand, what is the sequence of mRNA that would be transcribed? (Hint: Be sure to identify the initiation site.) 

ACUAUCUUCGUGAGAUG 

By examining the DNA sequence, we can see that there is a -10 consensus sequence near the 3’ end of the fragment. If we then count downstream, the +1 initiation site is the T immediately following the sequence AAT. This means the DNA fragment that will serve as the template for transcription has the sequence TGATAGAAGCACTCTAC. The mRNA made from this template will have complimentary base pairing with uracil (U) instead of thymine (T). This gives us ACUAUCUUCGUGAGAUG as the transcribed mRNA sequence. 

### Glossary

**consensus** DNA sequence that is used by many species to perform the same or similar functions

**core enzyme** prokaryotic RNA polymerase consisting of _α_, _α_, _β_, and _β_' but missing _σ_; this complex performs elongation

**downstream** nucleotides following the initiation site in the direction of mRNA transcription; in general, sequences that are toward the 3' end relative to a site on the mRNA

**hairpin** structure of RNA when it folds back on itself and forms intramolecular hydrogen bonds between complementary nucleotides

**holoenzyme** prokaryotic RNA polymerase consisting of _α_, _α_, _β_, _β_', and _σ_; this complex is responsible for transcription initiation

**initiation site** nucleotide from which mRNA synthesis proceeds in the 5' to 3' direction; denoted with a “+1”

**nontemplate strand** strand of DNA that is not used to transcribe mRNA; this strand is identical to the mRNA except that T nucleotides in the DNA are replaced by U nucleotides in the mRNA

**plasmid** extrachromosomal, covalently closed, circular DNA molecule that may only contain one or a few genes; common in prokaryotes

**promoter** DNA sequence to which RNA polymerase and associated factors bind and initiate transcription

**rho-dependent termination** in prokaryotes, termination of transcription by an interaction between RNA polymerase and the rho protein at a run of G nucleotides on the DNA template

**rho-independent** termination sequence-dependent termination of prokaryotic mRNA synthesis; caused by hairpin formation in the mRNA that stalls the polymerase

**TATA box** conserved promoter sequence in eukaryotes and prokaryotes that helps to establish the initiation site for transcription

**template strand** strand of DNA that specifies the complementary mRNA molecule

**transcription bubble** region of locally unwound DNA that allows for transcription of mRNA

**upstream** nucleotides preceding the initiation site; in general, sequences toward the 5' end relative to a site on the mRNA

   [1]: https://cnx.org/resources/4ca413565a4d6f1b242f35d7dcc74b91225e313a/Figure_15_02_02a.png
   [2]: https://cnx.org/resources/ece1845732f9f23f9990fd26082a48d7d159606b/Figure_15_02_01.jpg
   [3]: https://cnx.org/resources/c02143a9cbcf78aca6f670790432c6757806d8d9/Figure_15_02_03.jpg

