##  17.2 Mapping Genomes 

### Summary

By the end of this section, you will be able to do the following: 

  - Define genomics
  - Describe genetic and physical maps
  - Describe genomic mapping methods

**Genomics** is the study of entire genomes, including the complete set of genes, their nucleotide sequence and organization, and their interactions within a species and with other species. **Genome mapping** is the process of finding the locations of genes on each chromosome. The maps that genome mapping create are comparable to the maps that we use to navigate streets. A **genetic map** is an illustration that lists genes and their location on a chromosome. Genetic maps provide the big picture (similar to an interstate highway map) and use genetic markers (similar to landmarks). A **genetic marker** is a gene or sequence on a chromosome that co-segregates (shows genetic linkage) with a specific trait. Early geneticists called this linkage analysis. Physical maps present the intimate details of smaller chromosome regions (similar to a detailed road map). A **physical map** is a representation of the physical distance, in nucleotides, between genes or genetic markers. Both genetic linkage maps and physical maps are required to build a genome’s complete picture. Having a complete genome map of the genome makes it easier for researchers to study individual genes. Human genome maps help researchers in their efforts to identify human disease-causing genes related to illnesses like cancer, heart disease, and cystic fibrosis. We can use genome mapping in a variety of other applications, such as using live microbes to clean up pollutants or even prevent pollution. Research involving plant genome mapping may lead to producing higher crop yields or developing plants that better adapt to climate change.

### Genetic Maps

The study of genetic maps begins with **linkage analysis**, a procedure that analyzes the recombination frequency between genes to determine if they are linked or show independent assortment. Scientists used the term _linkage_ before the discovery of DNA. Early geneticists relied on observing phenotypic changes to understand an organism’s genotype. Shortly after Gregor Mendel (the father of modern genetics) proposed that traits were determined by what we now call genes, other researchers observed that different traits were often inherited together, and thereby deduced that the genes were physically linked by their location on the same chromosome. Gene mapping relative to each other based on linkage analysis led to developing the first genetic maps.

Observations that certain traits were always linked and certain others were not linked came from studying the offspring of crosses between parents with different traits. For example, in garden pea experiments, researchers discovered, that the flower’s color and plant pollen’s shape were linked traits, and therefore the genes encoding these traits were in close proximity on the same chromosome. We call exchanging DNA between homologous chromosome pairs **genetic recombination**, which occurs by crossing over DNA between homologous DNA strands, such as nonsister chromatids. Linkage analysis involves studying the recombination frequency between any two genes. The greater the distance between two genes, the higher the chance that a recombination event will occur between them, and the higher the recombination frequency between them. Figure shows two possibilities for recombination between two nonsister chromatids during meiosis. If the recombination frequency between two genes is less than 50 percent, they are linked.

![A homologous pair of chromosomes has three genes, named A, B, and C. Gene A is located near the top of the chromosome, and genes B and C are located close together near the bottom. Each chromosome has different A, B, and C alleles. The alleles may recombine if a crossover occurs between them, so that genetic material from one chromosome is swapped with another. Genes A and B are far apart on the chromosome such that a crossover event occurring almost anywhere in the chromosome will result in the recombination of alleles for these genes. Genes B and C are much closer together, so only crossovers occurring in a very narrow region will result in recombination of these genes.][1] _Crossover may occur at different locations on the chromosome. Recombination between genes _A_ and _B_ is more frequent than recombination between genes _B_ and _C_ because genes _A_ and _B_ are farther apart. Therefore, a crossover is more likely to occur between them._

The generation of genetic maps requires markers, just as a road map requires landmarks (such as rivers and mountains). Scientists based early genetic maps on using known genes as markers. Scientists now use more sophisticated markers, including those based on non-coding DNA, to compare individuals’ genomes in a population. Although individuals of a given species are genetically similar, they are not identical. Every individual has a unique set of traits. These minor differences in the genome between individuals in a population are useful for genetic mapping purposes. In general, a good genetic marker is a region on the chromosome that shows variability or polymorphism (multiple forms) in the population.

Some genetic markers that scientists use in generating genetic maps are **restriction fragment length polymorphisms** (RFLP), variable number of tandem repeats (VNTRs), **microsatellite polymorphisms**, and the **single nucleotide polymorphisms** (SNPs). We can detect RFLPs (sometimes pronounced “rif-lips”) when the DNA of an individual is cut with a restriction endonuclease that recognizes specific sequences in the DNA to generate a series of DNA fragments, which we can then analyze using gel electrophoresis. Every individual’s DNA will give rise to a unique pattern of bands when cut with a particular set of restriction endonucleases. Scientists sometimes refer to this as an individual’s DNA “fingerprint.” Certain chromosome regions that are subject to polymorphism will lead to generating the unique banding pattern. VNTRs are repeated sets of nucleotides present in DNA’s non-coding regions. Non-coding, or “junk,” DNA has no known biological function; however, research shows that much of this DNA is actually transcribed. While its function is uncertain, it is certainly active, and it may be involved in regulating coding genes. The number of repeats may vary in a population’s individual organisms. Microsatellite polymorphisms are similar to VNTRs, but the repeat unit is very small. SNPs are variations in a single nucleotide.

Because genetic maps rely completely on the natural process of recombination, natural increases or decreases in the recombination level given genome area affects mapping. Some parts of the genome are recombination hotspots; whereas, others do not show a propensity for recombination. For this reason, it is important to look at mapping information developed by multiple methods.

### Physical Maps

A physical map provides detail of the actual physical distance between genetic markers, as well as the number of nucleotides. There are three methods scientists use to create a physical map: cytogenetic mapping, radiation hybrid mapping, and sequence mapping. **Cytogenetic mapping** uses information from microscopic analysis of stained chromosome sections (Figure). It is possible to determine the approximate distance between genetic markers using cytogenetic mapping, but not the exact distance (number of base pairs). **Radiation hybrid mapping** uses radiation, such as x-rays, to break the DNA into fragments. We can adjust the radiation amount to create smaller or larger fragments. This technique overcomes the limitation of genetic mapping, and we can adjust the radiation so that increased or decreased recombination frequency does not affect it. **Sequence mapping **resulted from DNA sequencing technology that allowed for creating detailed physical maps with distances measured in terms of the number of base pairs. Creating **genomic libraries** and **complementary DNA (cDNA) libraries** (collections of cloned sequences or all DNA from a genome) has sped the physical mapping process. A genetic site that scientists use to generate a physical map with sequencing technology (a sequence-tagged site, or STS) is a unique sequence in the genome with a known exact chromosomal location. An **expressed sequence tag (EST)** and a single sequence length polymorphism (SSLP) are common STSs. An EST is a short STS that we can identify with cDNA libraries, while we obtain SSLPs from known genetic markers, which provide a link between genetic and physical maps.

![Cytogenetic maps of the 22 human autosomes and the X and Y chromosomes are shown. The map appears as a black, white, and gray banding pattern unique to each chromosome.][2] _A cytogenetic map shows the appearance of a chromosome after scientists stain and exam it under a microscope. (credit: National Human Genome Research Institute)_

### Genetic and Physical Maps Integration

Genetic maps provide the outline and physical maps provide the details. It is easy to understand why both genome mapping technique types are important to show the big picture. Scientists use information from each technique in combination to study the genome. Scientists are using genomic mapping with different model organisms for research. Genome mapping is still an ongoing process, and as researchers develop more advanced techniques, they expect more breakthroughs. Genome mapping is similar to completing a complicated puzzle using every piece of available data. Mapping information generated in laboratories all over the world goes into central databases, such as GenBank at the National Center for Biotechnology Information (NCBI). Researchers are making efforts for the information to be more easily accessible to other researchers and the general public. Just as we use global positioning systems instead of paper maps to navigate through roadways, NCBI has created a genome viewer tool to simplify the data-mining process.

Scientific Method Connection

How to Use a Genome Map Viewer 

Problem statement: Do the human, macaque, and mouse genomes contain common DNA sequences?

Develop a hypothesis.

To test the hypothesis, click this [link][3].

In Search box on the left panel, type any gene name or phenotypic characteristic, such as iris pigmentation (eye color). Select the species you want to study, and then press Enter. The genome map viewer will indicate which chromosome encodes the gene in your search. Click each hit in the genome viewer for more detailed information. This type of search is the most basic use of the genome viewer. You can also use it to compare sequences between species, as well as many other complicated tasks.

Is the hypothesis correct? Why or why not?

### Section Summary

Genome mapping is similar to solving a big, complicated puzzle with pieces of information coming from laboratories all over the world. Genetic maps provide an outline for locating genes within a genome, and they estimate the distance between genes and genetic markers on the basis of recombination frequencies during meiosis. Physical maps provide detailed information about the physical distance between the genes. The most detailed information is available through sequence mapping. Researchers combine information from all mapping and sequencing sources to study an entire genome.

### Critical Thinking Questions

Why is so much effort being poured into genome mapping applications?

Genome mapping has many different applications and provides comprehensive information that can be used for predictive purposes.

How could a genetic map of the human genome help find a cure for cancer?

A human genetic map can help identify genetic markers and sequences associated with high cancer risk, which can help to screen and provide early detection of different types of cancer.

### Glossary

**cytogenetic mapping** technique that uses a microscope to create a map from stained chromosomes

**expressed sequence tag (EST)** short STS that is identified with cDNA

**genetic map** outline of genes and their location on a chromosome

**genetic marker** gene or sequence on a chromosome with a known location that is associated with a specific trait

**genetic recombination** DNA exchange between homologous chromosome pairs

**genome mapping** process of finding the location of genes on each chromosome

**cDNA library** collection of cloned cDNA sequences

**genomic library** collection of cloned DNA which represents all of the sequences and fragments from a genome

**genomics** study of entire genomes including the complete set of genes, their nucleotide sequence and organization, and their interactions within a species and with other species

**linkage analysis** procedure that analyzes recombining genes to determine if they are linked

**microsatellite polymorphism** variation between individuals in the sequence and number of microsatellite DNA repeats

**physical map** representation of the physical distance between genes or genetic markers

**radiation hybrid mapping** information obtained by fragmenting the chromosome with x-rays

**restriction fragment length polymorphism (RFLP)** variation between individuals in the length of DNA fragments, which restriction endonucleases generate

**sequence mapping** mapping information obtained after DNA sequencing

**single nucleotide polymorphism (SNP)** variation between individuals in a single nucleotide

**variable number of tandem repeats (VNTRs)** variation in the number of tandem repeats between individuals in the population

   [1]: https://cnx.org/resources/b38ee4e5428bac208effc8fd0e799a786af7013c/Figure_17_02_01.jpg
   [2]: https://cnx.org/resources/61adf72f30deaa61ff8af8b6be34f7986d8b082d/Figure_17_02_02.jpg
   [3]: http://www.ncbi.nlm.nih.gov/mapview/

